pkg/testutils/lint/lint_test.go

```diff
@@ -1495,6 +1501,10 @@ func TestLint(t *testing.T) {
 				&lint.GlobIgnore{Pattern: "github.com/cockroachdb/cockroach/pkg/ui/embedded.go", Checks: []string{"S1013"}},
 			},
 			staticcheck.NewChecker(): {
+				// TODO(dt): remove when https://github.com/dominikh/go-tools/issues/430 is resolved.
+				ignoreSA4000{},
+				// We still support go 1.11 where regex.Copy avoids contention.
+				&lint.GlobIgnore{Pattern: "github.com/cockroachdb/cockroach/pkg/util/log/clog.go", Checks: []string{"SA1019"}},
```

Comment:
Once I fix https://github.com/dominikh/go-tools/issues/437 you'll be able to use the `-go 1.11` flag to not generate deprecation warnings for things that were deprecated in later versions.